### PR TITLE
configs: Add starqltechn

### PR DESCRIPTION
**Pull request by dsankouski**

### CODE REVIEW

File: configs/samsung-starqltechn [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	# GPU + IPA + CDSP, Venus, Wi-Fi
+	par apnhlos
+	fw image/a630_zap.{mdt,b*}     qcom/sdm845/starqltechn/a630_zap.mbn
+	fw -o image/a630_gmu.bin       qcom/
+	fw image/ipa_fws.{mdt,b*}      qcom/sdm845/starqltechn/ipa_fws.mbn
+	fw image/cdsp.{mdt,b*}         qcom/sdm845/starqltechn/cdsp.mbn
+	fw image/venus.{mdt,b*}        qcom/sdm845/starqltechn/venus.mbn
+	fw image/bdwlan*.*             bdwlan/
+	fw image/wlanmdsp.mbn          qcom/sdm845/starqltechn/
+	fw image/cdspr,slpi{r,us}}.jsn qcom/sdm845/starqltechn/
```
Comment (JamiKettunen) [inline configs/samsung-starqltechn]: `image/` should be `image/{`

File: configs/samsung-starqltechn [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+	# ADSP, Modem
+	par modem
+	fw image/adsp.{mdt,b*}           qcom/sdm845/starqltechn/adsp.mbn
+	fw image/modem.{mdt,b*}          qcom/sdm845/starqltechn/modem.mbn
+	fw image/{mba}.mbn               qcom/sdm845/starqltechn/
```
Comment (JamiKettunen) [inline configs/samsung-starqltechn]: `{mba}` should be `mba`, there's nothing to expand

File: configs/samsung-starqltechn [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	fw image/{adsp{r,ua},modem{r,uw} qcom/sdm845/starqltechn/
```
Comment (JamiKettunen) [inline configs/samsung-starqltechn]: Missing `}.jsn` before the target dir?

**Review by JamiKettunen — CHANGES_REQUESTED** [review]
Thanks! However looks untested ;)

**Comment by JamiKettunen** [comment]
Fwiw I'm planning to deprecate this project later once https://gitlab.com/mobian1/droid-juicer gains all the features this already has currently, so it may make sense to send in a config there as well (I should do so with the devices I can test, will take a look later tonight)

File: configs/samsung-starqltechn (hunk 1 above)
Comment (dsankouski) [reply configs/samsung-starqltechn]: fixed

File: configs/samsung-starqltechn (hunk 3 above)
Comment (dsankouski) [reply configs/samsung-starqltechn]: fixed

File: configs/samsung-starqltechn (hunk 2 above)
Comment (dsankouski) [reply configs/samsung-starqltechn]: fixed

**Comment by dsankouski** [comment]
fixed comments, reformatted, tested on device

**Pull request merged**